TROUBLESHOOTING.md

```diff
@@ -74,7 +74,7 @@ We follow the MAJOR.MINOR.PATCH versioning scheme as described by [SemVer.org](h
 <a name="environment"></a>
 ## Environment Variables and Your SendGrid API Key
 
-All of our examples assume you are using [environment variables](https://github.com/sendgrid/sendgrid-nodejs#setup-environment-variables) to hold your SendGrid API key.
```

Comment:
I think we should use a relative link here instead so that GH always transforms the link properly regardless of the branch someone may be on.

Reply:
Thank you for the feedback. Made the change to use a relative link.